src/index.css

```diff
@@ -32,3 +32,7 @@ h1 {
 .nav {
   margin-bottom: 10px;
 }
+
+.ms-List-cell a{
```

Comment:
```suggestion
.ms-Link {
```

I think it's better to use `.ms-Link` selector

Reply:
thanks for your feedback!